docs/content/tutorials/setup.md

```diff
@@ -40,6 +40,13 @@ We will use [Docker][docker-docs] to accomplish this.
 
 8. Navigate your browser to the url you chose in step 6. Your instance is now ready to use.
 
+!!! info Using a different port
+    This tutorial assumes that you run your HedgeDoc 2 instance on port 80 and 443 (HTTP and HTTPS).
+    If you want to use a custom port, be sure to update your `.env` file to include the port in the
+    `HD_BASE_URL` variable as well as update the port bindings in the `docker-compose.yml` file.
+    For example, when using port 8080 instead, use the string `8080:8080` (both sides) in the ports
```

Comment:
instead of giving a port as example, maybe give an example base url?

Reply:
Okay